scripts/Modules/mkw_core.py

```diff
@@ -88,3 +88,31 @@ def get_speed(playerIdx=0):
     xyz = math.sqrt(x ** 2 + y ** 2 + z ** 2)
 
     return speed(x, y, z, xz, xyz)
+
+# Next 3 functions are used for exact finish display
+def getIGT(lap, player): # lap is 1-indexed
```

Comment:
I would recommend a PEP 257 docstring here instead of a comment. Modern IDEs will display it nicely to developers.